Frontend/Services/CreateHtbDocument.cs

```diff
@@ -190,6 +190,24 @@ private static void Rationale(DocumentBuilder builder, Project project)
 
             builder.AddParagraph(pBuilder => pBuilder.AddText(project.Rationale.Project));
 
+            builder.AddNumberedList(lBuilder =>
```

Comment:
Just flagging this in case these need to be removed or changed as they will appear on htb doc e.g. if we are doing a demo. But it may be intentional to demonstrate lists etc. so just flagging.